templates/default/nginx.conf.erb

```diff
@@ -36,52 +34,51 @@ server {
+              style.sheet.insertRule(".global-nav, .app-wrapper, .chrHeaderWrapper, .euiNavDrawer { margin-top: 42px !important }", 0);
 
-                let banner = document.createElement("div");
-                banner.innerHTML = "]] .. string.format("<%= @message_format %>", app_names_message) .. [[";
-                banner.style.backgroundColor = "red";
-                banner.style.color = "white";
-                banner.style.fontFamily = "Arial, Helvetica, sans-serif";
-                banner.style.fontSize = "22px";
-                banner.style.lineHeight = "1";
-                banner.style.overflowX = "scroll";
-                banner.style.padding = "10px";
-                banner.style.position = "fixed";
-                banner.style.textAlign = "center";
-                banner.style.top = "0";
-                banner.style.whiteSpace = "nowrap";
-                banner.style.width = "100%";
-                banner.style.zIndex = "1000";
+              let banner = document.createElement("div");
+              banner.innerHTML = "]] .. string.format("<%= @message_format %>", app_names_message) .. [[";
+              banner.style.backgroundColor = "red";
+              banner.style.color = "white";
+              banner.style.fontFamily = "Arial, Helvetica, sans-serif";
+              banner.style.fontSize = "22px";
+              banner.style.lineHeight = "1";
+              banner.style.overflowX = "scroll";
+              banner.style.padding = "10px";
+              banner.style.position = "fixed";
+              banner.style.textAlign = "center";
+              banner.style.top = "0";
+              banner.style.whiteSpace = "nowrap";
+              banner.style.width = "100%";
+              banner.style.zIndex = "1000";
 
-                document.body.appendChild(banner);
-                let observer = new MutationObserver(function(mutations) {
-                  for (let mutation of mutations) {
-                    for (let removedNode of mutation.removedNodes) {
-                      if (removedNode == banner) {
-                        document.body.appendChild(banner);
-                      }
+              document.body.appendChild(banner);
+              let observer = new MutationObserver(function(mutations) {
+                for (let mutation of mutations) {
+                  for (let removedNode of mutation.removedNodes) {
+                    if (removedNode == banner) {
+                      document.body.appendChild(banner);
                     }
                   }
-                });
-                observer.observe(document.body, { childList: true })
-              </script>
+                }
+              });
+              observer.observe(document.body, { childList: true });
             ]]);
           end
         end
       end
-      ngx.say("</html>")
+
+      ngx.say([[
+        console.log("Barito-Flavor: Customization for Kibana has been loaded.");
```

Comment:
is it for debugging purpose? can we remove this?

Reply:
Yes, but it will speed up our discovery whether is the script is being loaded or not. Previously we have an issue with the new Kibana which doesn't load inline scripts at all.

Reply:
Based on our discussion on Zoom, we'll remove this.